docs/sources/alerting/monitor-status/view-alert-rules.md

```diff
@@ -76,3 +76,22 @@ To view or restore previous versions for an alert rule, complete the following s
 
 On the Alert rule's Versions page you can view, compare and restore the previous rule versions.
 {{< figure src="/media/docs/alerting/screenshot-grafana-alerting-version-history-v3.png" max-width="750px" alt="View alert rule history to compare and restore previous alert rules." >}}
+
+## Bulk pause or resume alert rules evaluations within a folder
+
+Admin users can pause or resume all of the alert rules evaluations within a folder. To pause or resume all the alert rules evaluations in a folder, click the menu icon and select **Pause all rule evaluation** or **Resume all rule evaluation**.
+
+## Bulk delete all alert rules within a folder
+
+Admin users can delete all of the alert rules within a folder. To delete all the alert rules in a folder, click the menu icon and select **Delete**. Then type "Delete" into the field and click **Delete** to confirm the bulk deletion.
+
+## Permanently delete or restore deleted alert rules
```

Comment:
I think we could move this entry to a new section belllow monitor-status (`Monitor` alerts section). wdyt? @JohnnyK-Grafana